yapapi/executor/__init__.py

```diff
@@ -510,7 +522,111 @@ async def accept_payment_for_agreement(agreement_id: str, *, partial: bool = Fal
 
         storage_manager = await self._stack.enter_async_context(gftp.provider())
 
+        def unpack_work_item(item: WorkItem) -> Tuple[Work, ExecOptions]:
+            """Extract `Work` object and options from a work item.
+            If the item does not specify options, default ones are provided.
+            """
+            if isinstance(item, tuple):
+                return item[0], item[1]
```

Comment:
isn't this the same as just
```suggestion
                return item
```